skimage/restoration/_denoise.py

```diff
@@ -478,8 +490,8 @@ def _wavelet_threshold(img, wavelet, threshold=None, sigma=None, mode='soft',
     return pywt.waverecn(denoised_coeffs, wavelet)[original_extent]
 
 
-def denoise_wavelet(img, sigma=None, wavelet='db1', mode='soft',
-                    wavelet_levels=None, multichannel=False,
+def denoise_wavelet(img, sigma=None, method='BayesShrink', wavelet='db1',
+                    mode='soft', wavelet_levels=None, multichannel=False,
```

Comment:
I think `method` should most likely be added as the last argument here to avoid breaking existing code.